src/utils/config.ts

```diff
@@ -10,6 +10,8 @@ const ciConfig = {
   commitMessage: process.env.CI_COMMIT_MESSAGE,
   forgeType: process.env.CI_FORGE_TYPE,
   githubToken: process.env.PLUGIN_GITHUB_TOKEN,
+  giteaToken: process.env.PLUGIN_GITEA_TOKEN,
```

Comment:
Why not just deprecate GitHub token env and use `token` as there already is forge type so no point in different forge token settings

Reply:
and as woodpecker do tell the plugin already what forge type it is connected to ... we can just deprecate `forgeType` :)

Reply:
Will adjust the token name.

It already using woodpeckers forge type: `forgeType: process.env.CI_FORGE_TYPE` 
Nothing we need to deprecate there.